src/browser/of_events.ts

```diff
@@ -33,6 +34,7 @@ class OFEvents extends EventEmitter {
         if (tokenizedRoute.length >= 2) {
             const [channel, topic] = tokenizedRoute;
             const uuid: string = (payload && payload.uuid) || tokenizedRoute[2] || '*';
+            const name: string|false = (payload && payload.name);
```

Comment:
Mostly irrelevant nitpick: will eval to `undefined` if there's no payload or no name.
`(!!payload && payload.name)` will evaluate to `false` if there's no payload, but I think unidentified is probably actually what we want in this case anyhow.

Reply:
Agreed. Is there a case in which this could not work as expected?

Reply:
I don't think so. I would remove the `|false` typing, since this will never eval to `false`. But code looks good as is.

Reply:
Ah, right, good call, missed that. Changed.